11-node-frameworks/classworks/Volodkova_Anna/index.js

```diff
@@ -0,0 +1,55 @@
+import {incCounter, schema} from "./functions";
+
+const koa = require('koa');
+const serve = require('koa-static');
+const session = require('koa-generic-session');
+const bodyParser = require('koa-bodyparser');
+const Router = require('koa-router');
+
+
+const app = new koa();
+
+app.use(bodyParser());
+
+app.use(serve(`public`));
+
+app.use(session());
+const inc = incCounter(session);
```

Comment:
Please, use these functions as mindware, instead of calling it in `/summer` endpoint

Reply:
Please, explain this point in more detail

Reply:
You need to make it works everywhere by default without adding additional call to every new endpoint. So I suggest adding separate middleware which will be count sessions views and transfer the call further. Like body parser does. When you use body-parser you don't need to call it for each body, it happens automatically. Do the same, and store result in session